src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlCachedBuffer.cs

```diff
@@ -140,7 +138,7 @@ public bool IsNull
         {
             get
             {
-                return (_cachedBytes == null) ? true : false;
+                return (_cachedBytes == null);
```

Comment:
You can probably make this and a few others expression bodied properties. If the getter is a single expression or a simple ternary so it's clear from a single line of text that it doesn't I usually do so. 
I think my rule of thumb is if I feel I'd need a line break or it's likely to need scrolling to see the end of the expression I'll leave it as statement body. See what you think.

Reply:
Thanks! I've add a new commit to change those getter accessors to single line expression bodied definitions.